### PR TITLE
PMM-7552: add Percona Docker label

### PR DESCRIPTION
**Pull request by nikita-b**

### CODE REVIEW

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1851-e4ee0b6
client docker - perconalab/pmm-client-fb:PR-1851-e4ee0b6
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1851-e4ee0b6.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1851-e4ee0b6&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1851-e4ee0b6.tar.gz

**Comment by nikita-b** [comment]
New container labels:
```
"Labels": {
                "org.label-schema.build-date": "2021-06-16 11:42:01+00:00",
                "org.label-schema.license": "AGPL-3.0",
                "org.label-schema.name": "Percona Monitoring and Management",
                "org.label-schema.schema-version": "1.0",
                "org.label-schema.vendor": "Percona",
                "org.label-schema.version": "PR-1851-e4ee0b6",
                "org.opencontainers.image.created": "2021-06-16 11:42:01+00:00",
                "org.opencontainers.image.licenses": "AGPL-3.0",
                "org.opencontainers.image.title": "Percona Monitoring and Management",
                "org.opencontainers.image.vendor": "Percona",
                "org.opencontainers.image.version": "PR-1851-e4ee0b6"
            }
```

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/7213/

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1851-e4ee0b6
client docker - perconalab/pmm-client-fb:PR-1851-e4ee0b6
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1851-e4ee0b6.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1851-e4ee0b6&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1851-e4ee0b6.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/7221/

**Review by ademidoff — APPROVED** [review]
👍

File: build/pmm-server-docker/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
+LABEL org.label-schema.license AGPL-3.0
+LABEL org.label-schema.name Percona Monitoring and Management
+LABEL org.label-schema.vendor Percona
+LABEL org.label-schema.version ${VERSION}
+LABEL org.opencontainers.image.created ${BUILD_DATE}
+LABEL org.opencontainers.image.licenses AGPL-3.0
+LABEL org.opencontainers.image.title Percona Monitoring and Management
+LABEL org.opencontainers.image.vendor Percona
+LABEL org.opencontainers.image.version ${VERSION}
+
```
Comment (ademidoff) [inline build/pmm-server-docker/Dockerfile:16]: oops, looks like `"org.label-schema.schema-version" is set to "1.0" by the upstream image. Maybe we can set to `2.x` meaning that it has to do with PMM v2?

Comment (nikita-b) [reply build/pmm-server-docker/Dockerfile:16]: `schema-version` is version of schema: http://label-schema.org/rc1/
It's just standard version like http 2.0 :)

**Pull request merged**